code/__DEFINES/cult_defines.dm

```diff
@@ -66,3 +63,7 @@
 #define NARSIE_NEEDS_SUMMONING 2
 #define NARSIE_HAS_RISEN 3
 #define NARSIE_HAS_FALLEN -1
+
+#define GET_CULT_DATA(variable, default) (SSticker.cult_data ? SSticker.cult_data.variable : default)
```

Comment:
Can you autodoc this, and maybe rename it to something more descriptive like get cult static data? Mostly so it's more obvious about what it's doing (vs just pulling from the datum)

Reply:
I'm not sure if its technically really static, you can call procs on it, such as its `get_icon` method. I'll add autodoc though